modules/books.js

```diff
@@ -0,0 +1,7 @@
+function Books(id, title, author) {
```

Comment:
- [ ] Good work creating a module for Books. Remember that for naming conventions we use names starting with uppercase for classes, this `Books` it's a class. Kindly change this function to a class to follow best practices and avoid future mistakes or confusion